core/shared/utils/bh_time.h

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (C) 2019 Intel Corporation.  All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ */
+
+#ifndef _BH_TIME_
+#define _BH_TIME_
+#include "bh_platform.h"
```

Comment:
why do we have to include this file?

Reply:
In the bh_platform.h is quite similar to platform_common.h and connects the header files to use typedef like unit64 which bh_time.c needs. All the files have bh_platform.h in their header files.